src/test/java/API_hw2/TrelloApiTest.java

```diff
@@ -0,0 +1,81 @@
+package API_hw2;
+
+import API_hw2.constants.Length;
+import API_hw2.constants.ResponseStatus;
+import API_hw2.core.TrelloServiceObj;
+import API_hw2.beans.TrelloBoard;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.testng.annotations.Test;
+
+import static org.hamcrest.Matchers.*;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.nullValue;
+
+public class TrelloApiTest {
```

Comment:
In many tests you first create a board, then delete it. Remove the methods to @BeforeMethod/@AfterMethod to avoid code duplication

Reply:
If you don't want your Before/After run before and after each method - use groups

Reply:
Added @BeforeMethod, @AfterMethod with groups